src/Foundation/People/platform/Extenders/CustomSchemaExtender.cs

```diff
@@ -35,13 +43,23 @@ public CustomSchemaExtender()
 		private string GetLatestMvpYear(Sitecore.Data.Items.Item personItem)
 		{
 			var awards = GetAwardsTags(personItem);
-			if(awards.Any())
+			if (awards.Any())
 			{
 				return awards.First().Key.ToString();
 			}
 			return "";
 		}
 
+		private string GetLatestMvpYearList(Sitecore.Data.Items.Item personItem)
```

Comment:
@joseamneto please rename the function name as it is quite confusing, the name does not reflect what is returning.

Reply:
function renamed to GetAwardsCount